.github/workflows/python-publish.yml

```diff
@@ -5,11 +5,26 @@ name: Upload Python Package
 
 on:
   workflow_dispatch:
-  release:
-    types: [published]
+  push:
+    branches:
+    - main
 
 jobs:
+  tagpr:
+    permissions:
+      actions: write
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@96f53100ba2a5449eb71d2e6604bbcd94b9449b5
+    - uses: Songmu/tagpr@43d52e123cf8d55db9d602601f115f530588e2f8
```

Comment:
Could you comment these actions version?

Reply:
[08bcc0c](https://github.com/launchableinc/cli/pull/595/commits/08bcc0c9b7641ce9330eaff01dca1c688e8b858c)